### DISCUSSION TITLE
Adding second Storage Array Error

### DISCUSSION THREAD

**Discussion by falarcon10** [NetApp/harvest#2997] [Q&A] [answered]
Hi!
I'm installing NAbox v4.01 with Harvest 24.06.17 and I can add first Ontap System, when I go to register another one, This error appears "error while adding system empty cluster name" There are two MCC-IP nodes, from SITE-A and SITE-B. The MCC works right and the two nodes have their own name.
What could be the problem?
Thanks

**Comment by cgrinds** [comment]
Can you share your log files @falarcon10?
@ybizeul do you have updated documentation on gathering logs for NAbox 4?
If there are no new instructions, you can follow these @falarcon10 to gather the logs.
https://netapp.github.io/harvest/nightly/help/log-collection/

**Reply by falarcon10** [reply]
Thanks @cgrinds, 
I am not able to obtain the logs as explained in the link you have indicated.
I'll keep trying
Thanks

**Reply by cgrinds** [reply]
Try running this on your nabox4 instance after sshing into it.
`dc logs havrest > havrest.txt`

**Reply by falarcon10** [reply]
Thanks, now yes.
The node that was add ok is ESSTOFPNAPC1C01 and the node with problems is ESSTOFPNAPC2C01. If I delete ESSTOFPNAPC1C01 and start again with node ESSTOFPNAPC2C01 the error is the same. I revise the MCC nodes and I can't see any differences.
[havrest.txt](https://github.com/user-attachments/files/15872015/havrest.txt)

**Reply by cgrinds** [reply, marked as answer]
Thanks @falarcon10 ! Harvest works fine with ZAPIs disabled, but I think nabox4 is failing to detect the ontap version when you add the cluster when ZAPIs are disabled on that cluster. @ybizeul is going to take a look.

havrest  | time=2024-06-17T13:02:27.549Z level=DEBUG source=ontapInfo.go:167 msg="ontap response" text="<?xml version='1.0' encoding='UTF-8' ?>\n<!DOCTYPE netapp SYSTEM 'file:/etc/netapp_gx.dtd'>\n<netapp version='1.241' xmlns='[http://www.netapp.com/filer/admin'>\n](http://www.netapp.com/filer/admin%27%3E/n) <results reason=\"ONTAPI has been proactively suspended on this cluster due to inactivity as this feature is deprecated. Use the &quot;system services web ontapi modify&quot; CLI command to un-suspend ONTAPI.\" status=\"failed\" errno=\"61253\"/>\n</netapp>"
 
havrest  | time=2024-06-17T13:02:27.550Z level=ERROR source=error.go:21 msg="empty cluster name"

**Reply by falarcon10** [reply]
That's correct. 
In configured cluster the ontapi suspended is FALSE and in the <failed> node is TRUE
Best regards.

**Reply by ybizeul** [reply]
In the info box in NAbox UI you can collect a support bundle.

**Reply by ybizeul** [reply]
What is the string returned by "version" in ONTAP ?

**Reply by falarcon10** [reply]
This is the string "NetApp Release 9.14.1P5: Sat May 18 19:42:16 UTC 2024"
With "ontapi suspended is FALSE" on both nodes, two nodes can add to NAbox.
Thanks
